packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -103,168 +152,296 @@ class UpdatePackagesCommand extends FlutterCommand {
   }
 }
 
+// The various sections of a pubspec.yaml file. We care about the
+// "dependencies", "dev_dependencies", and "dependency_overrides" sections, the
+// others are all bucketed into "other".
```

Comment:
nit: use `///`, the style guide [encourages it](https://www.dartlang.org/guides/language/effective-dart/documentation#consider-writing-doc-comments-for-private-apis) for private things too and they support `[...]` links in IDEs.

(this comment applies throughout the PR)

Reply:
done